src/components/Create/Create.jsx

```diff
@@ -0,0 +1,94 @@
+import "./create.css"
+import React, { useState, useEffect } from 'react'
+
+export const Create = ({
+  addNewPost, 
+  fetchPosts, 
+  newPost, 
+  setNewPost}) => {
+
+    const [errorMessage, setErrorMessage] = useState("")  
+    const postAPI = "https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts"
+  
+  //------------Rendering error message if message is too long!----
+  useEffect(()=> {
+    if(newPost.length >= 141) {
```

Comment:
I believe this should have been set to ">= 140", as it is right now the text is red when the user has written 140 characters, but they can still send the "thought". The API is restricting messages longer than 140 :)